### ISSUE TITLE
why do you not use norm when evaluated zero-shot ?

### ISSUE THREAD

**Issue by Johnson-yue** [zer0int/CLIP-fine-tune#8]
https://github.com/zer0int/CLIP-fine-tune/blob/6834074a007c60de712f0bbaca0d80aad051a10c/ft-D-eval-model-on-my-finetune-dataset.py#L84

In train process,  using norm then compute cosine similarity, but in eval not used norm , is right?

**Comment by zer0int** [comment]
Oh, you're right - thank you for catching that! We should normalize the vector as magnitude is irrelevant / misleading. I adjusted the code accordingly.

**Comment by Johnson-yue** [comment]
you can close this issue when you updated the code
